### DISCUSSION TITLE
[List Page] Possibility of UI elements group skipping via Keyboard keys

### DISCUSSION THREAD

**Discussion by raghav-swankier** [UI5/webcomponents-react#6052] [Q&A]
Hi All,

Is there a possibility of skipping grouped UI elements when accessing the app through keyboard? For example, in the attached screenshot, can I skip group 2 and jump directly to group 3 via keyboard because if the user directly wants to access table, he has to go through each filters in the filter bar (group 2) which is unnecessary? This becomes time consuming if there are more filters. I couldn't find any documentation related to this. Please help if you have any documentation on this.

<img width="1719" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/121158363/ca25952c-062f-4076-bc5f-4b578017b64c">

Thanks you :)

**Comment by Lukas742** [comment]
Hi @I506645 

yup, this is possible with the [F6 Navigation](https://sap.github.io/ui5-webcomponents/docs/advanced/using-features/#f6-navigation-fast-navigation) feature. 

I've also created an example for this [issue](https://github.com/SAP/ui5-webcomponents-react/issues/6026), maybe it helps you as well: https://stackblitz.com/edit/github-zmyar5-zmrhkz?file=src%2FTableData.tsx,src%2Fmain.tsx,src%2FFilterFile.tsx

**Comment by raghav-swankier** [comment]
Thanks @Lukas742 , that works :)
